types/audio-play/audio-play-tests.ts

```diff
@@ -0,0 +1,6 @@
+import { audioPlay } from "audio-play";
```

Comment:
The import corresponding to the change I suggested above (and, as far as I can tell, the underlying library) is `import audioPlay = require("audio-play");`.

Reply:
I can also use `import audioPlay from "audio-play";` since using `export = audioPlay;` is essentially a default export?

Reply:
Same as above. The module-function template had all the answers.

Reply:
I believe it only functions as a default if you enable synthetic default exports in your tsconfig file.